assets/src/containers/ResourcesAccessed.js

```diff
@@ -250,23 +250,22 @@ function ResourcesAccessed (props) {
                   onChange={handleResourceGradeFilter}
                   inputProps={{
                     name: 'grade',
-                    id: 'grade-range',
+                    id: 'grade-range'
                   }}
                 >
-                  <MenuItem value="All">All</MenuItem>
-                  <MenuItem value="90-100">90-100%</MenuItem>
-                  <MenuItem value="80-89">80-89%</MenuItem>
-                  <MenuItem value="70-79">70-79%</MenuItem>
+                  <MenuItem value='All'>All</MenuItem>
+                  <MenuItem value='90-100'>90-100%</MenuItem>
+                  <MenuItem value='80-89'>80-89%</MenuItem>
+                  <MenuItem value='70-79'>70-79%</MenuItem>
                 </Select>
               </FormControl>
               {showSaveSetting
-                ?
-                  <Checkbox
-                    checked={saveSettingClicked}
-                    onChange={() => setSaveSettingClicked(!saveSettingClicked)}
-                    value='checked'
-                    color='primary'/>
-              : <div style={{ padding: '10px' }}></div>
+                ? <Checkbox
+                  checked={saveSettingClicked}
+                  onChange={() => setSaveSettingClicked(!saveSettingClicked)}
+                  value='checked'
+                  color='primary' />
```

Comment:
This seems to be a weird case where `standard` wants the `/>` to be both aligned with the tag beginning and to be un-indented once before it. Was that your experience? Is this a known problem with `standard`?

Reply:
For the `/>` I've been putting it on a new line if the component is multi-line. For example:
```js
<Checkbox
  checked={saveSettingClicked}
  onChange={() => setSaveSettingClicked(!saveSettingClicked)}
  value='checked'
  color='secondary'
/>
```
If the component is one-line, then the closing tag goes at the end with a space.
```js
<div style={{ padding: '10px' }} />
```